src/ui/CategorySelector/CategorySelector.svelte

```diff
@@ -137,6 +149,10 @@
       &--previous {
         background: url(./chevron--left.svg) no-repeat 0 50%;
       }
+      &--single-previous {
+        border-right: 1px solid rgba($color-white, 0.5);
+        background: url(./chevron--left.svg) no-repeat 0% 50%;
+      }
```

Comment:
Instead of creating a new modifier it would be better to just move the border from `&--next` to `&--previous` (left to right). That would stop the line jumping by 1px as well.

Reply:
Good shout, yeah I'll do that instead!